lib/src/view/opening_explorer/opening_explorer_screen.dart

```diff
@@ -0,0 +1,845 @@
+                                  options,
+                                  boardSize,
+                                  isTablet: isTablet,
+                                ),
+                              ],
+                            ),
+                          ),
+                          Flexible(
+                            fit: FlexFit.loose,
+                            child: Column(
+                              mainAxisAlignment: MainAxisAlignment.start,
+                              children: [
+                                Expanded(
+                                  child: PlatformCard(
+                                    margin: const EdgeInsets.all(
+                                      kTabletBoardTableSidePadding,
+                                    ),
+                                    semanticContainer: false,
+                                    child: _OpeningExplorer(
+                                      pgn: pgn,
+                                      options: options,
+                                    ),
+                                  ),
+                                ),
+                              ],
+                            ),
+                          ),
+                        ],
+                      )
+                    : Column(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: [
+                          if (isTablet)
+                            Padding(
+                              padding: const EdgeInsets.all(
+                                kTabletBoardTableSidePadding,
+                              ),
+                              child: AnalysisBoard(
+                                pgn,
+                                options,
+                                boardSize,
+                                isTablet: isTablet,
+                              ),
+                            )
+                          else
+                            AnalysisBoard(
+                              pgn,
+                              options,
+                              boardSize,
+                              isTablet: isTablet,
+                            ),
+                          Expanded(
+                            child: _OpeningExplorer(pgn: pgn, options: options),
+                          ),
+                        ],
+                      );
+              },
+            ),
+          ),
+        ),
+        _BottomBar(pgn: pgn, options: options),
+      ],
+    );
+  }
+}
+
+class _OpeningExplorer extends ConsumerWidget {
+  const _OpeningExplorer({
+    required this.pgn,
+    required this.options,
+  });
+
+  final String pgn;
+  final AnalysisOptions options;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final ctrlProvider = analysisControllerProvider(pgn, options);
+    final position = ref.watch(ctrlProvider.select((value) => value.position));
```

Comment:
I think we should not use all the `select()` here. As explained in [riverpod doc](https://riverpod.dev/docs/advanced/select#filtering-widgetprovider-rebuild-using-select), it's not necessary here and increase complexity.